### PR TITLE
Update database docker commands to create database at container startup

### PR DESCRIPTION
**Pull request by thomasdelhomenie**

### CODE REVIEW

File: content/development/test.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@ Please refer to the https://hub.docker.com/_/postgres/[Docker hub page] for full
 === Oracle
 
 A Docker image for Oracle database has been created and pushed to https://hub.docker.com/r/exoplatform/oracle/[Docker hub] for testing purpose.
-Here is a command example to start an Oracle 12cR2 database exposed on the default port (1521) :
+Here is a command example to start an Oracle 12cR2 database server exposed on the default port (1521) and creates a database `plf` :
```
Comment (vsellier) [inline content/development/test.adoc]: It's a detail but the plf database is already created on the *_plf oracle images. WDYT to replace "creates" by "starts" ?

Comment (thomasdelhomenie) [reply content/development/test.adoc]: I get your point, but "starts" for the fact that a database is already created is weird IMO. What about saying "Here is a command example to start an Oracle 12cR2 database server exposed on the default port (1521) with a database `plf`" ?

Comment (vsellier) [reply content/development/test.adoc]: 👍

Comment (thomasdelhomenie) [reply content/development/test.adoc]: Done

**Pull request merged**